flocker/acceptance/test_environment_variables.py

```diff
@@ -166,19 +170,13 @@ def mysql_can_connect():
                     try:
                         return connect(
                             host=host,
-                            port=MYSQL_EXTERNAL_PORT,
+                            port=port,
                             user=user,
                             passwd=passwd,
                             db=db,
                         )
-                    except OperationalError as e:
-                        # PyMySQL doesn't provided a structured way to
-                        # get this.
-                        # https://github.com/PyMySQL/PyMySQL/issues/274
-                        if "Connection refused" in str(e):
-                            return False
-                        else:
-                            raise
+                    except Error:
```

Comment:
Maybe at this point the error can be logged and then we can check the logs if it becomes a problem that we're returning `False` on all errors.